### PR TITLE
Replaced urllib with Requests

### PR DESCRIPTION
**Pull request by GavinFreud**
Title says it all. This seemed a lot simpler than it should be, any feedback and/or errors?

### CODE REVIEW

File: pittAPI.py [hunk 1]
```diff
@@ -412,9 +413,9 @@ def get_person(self, query, maxPeople=10):
 
                 if personurl.lower().startswith("https://") and hasattr(ssl, '_create_unverified_context'):
                     ct = ssl._create_unverified_context()
-                    f = urlopen(personurl, context=ct)
+                    f = s.get(personurl, context=ct)
```
Comment (RitwikGupta) [inline pittAPI.py]: In requests, there is no `context` arg for the `get()` method. This results in an error. Please look up the proper way to disable SSL verification in requests!

File: pittAPI.py [hunk 2]
```diff
@@ -411,10 +412,9 @@ def get_person(self, query, maxPeople=10):
...
 
                 if personurl.lower().startswith("https://") and hasattr(ssl, '_create_unverified_context'):
-                    ct = ssl._create_unverified_context()
-                    f = urlopen(personurl, context=ct)
+                    f = s.get(personurl, verify=False)
                 else:
-                    f = urlopen(personurl)
+                    f = s.get(personurl)
 
                 html = f.read()
```
Comment (RitwikGupta) [inline pittAPI.py]: We now have an error here, `AttributeError: 'Response' object has no attribute 'read'`. Please confirm on how to get the data for a Response object.

**Comment by GavinFreud** [comment]
Oh, sorry about that. It should now work by Requests' standards.

**Comment by RitwikGupta** [comment]
Awesome! Thank you for the great work 😁

**Pull request merged**